pkg/payload/payload_test.go

```diff
@@ -123,3 +123,52 @@ func mustRead(path string) []byte {
 	}
 	return data
 }
+
+func TestExclude(t *testing.T) {
+	tests := []struct {
+		exclude     string
+		profile     string
+		annotations map[string]interface{}
+
+		isExcluded bool
+	}{
+		{
+			exclude:     "",
+			annotations: map[string]interface{}{"exclude.release.openshift.io/identifier": "true"},
+		},
+		{
+			exclude:     "identifier",
+			annotations: map[string]interface{}{"exclude.release.openshift.io/identifier": "true"},
+			isExcluded:  true,
+		},
+		{
+			profile:     "crc",
+			annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"},
+			isExcluded:  true,
+		},
+		{
+			profile:     "edge",
+			annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"},
+		},
+		{
+			profile: "default",
```

Comment:
add a test case where `profile: "default"` shows us excluding `annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"}`, i.e. that the manifest only belongs implicitly to `default` if it declares no explicit profile inclusion.

Reply:
done :+1: